b2sdk/bucket.py

```diff
@@ -292,6 +292,36 @@ def list_parts(self, file_id, start_part_number=None, batch_size=None):
         """
         return self.api.list_parts(file_id, start_part_number, batch_size)
 
+    def list_file_versions(self, file_name, fetch_count=None):
+        """
+        Lists all of the versions for a single file.
+
+        Depending on a Bucket's settings many versions may exist for the same filename.
+        This lists all of the versions present in the Bucket.
+
+        :param str file_name: the name of the file to list.
+        :param int,None fetch_count: how many entries to list per API call or ``None`` to use the default. Acceptable values: 1 - 1000
+        :rtype: generator[b2sdk.v1.FileVersionInfo]
+        :returns: generator of file_version_info
+        """
+        start_file_id = None
+        session = self.api.session
+        while True:
```

Comment:
```suggestion
        while 1:
```
since we still support python 2.7. It's a long story :)

Reply:
Interesting. I see that the ls function also uses `while True:`, is that an issue too?

Reply:
It is not something I would go back and change but when I am reviewing newly added code, I flag this as a potential inefficiency. Fortunately we are not going to support 2.7 forever, so this should not be a problem for long.